tests/async_engine/test_async_llm_engine.py

```diff
@@ -139,7 +139,7 @@ def start_engine():
 
     return AsyncLLMEngine.from_engine_args(
         AsyncEngineArgs(model="facebook/opt-125m",
-                        enforce_eager=True,
+                        enforce_eager=False,
```

Comment:
Do we eliminate this way some eager tests ?  or is it used only in prepared for torch compile?